src/resources/postcss/components/full/_ical-link.pcss

```diff
@@ -14,27 +14,26 @@
 		display: inline-flex;
 		font-weight: var(--font-weight-bold);
 		justify-content: center;
-
-		&:before {
-			background-image: svg-inline(plus-focus);
-			background-repeat: no-repeat;
-			background-size: contain;
-			content: '';
-			display: inline-block;
+		
+		.tribe-events-c-ical__link-icon-svg {
 			flex: none;
 			height: 10px;
-			margin-right: var(--spacer-1);
 			width: 10px;
+			margin-right: var(--spacer-1);
+			
+			path {
+				stroke: var(--color-icon-focus);
+			}
 		}
```

Comment:
let's pull this out of `.tribe-events-c-ical__link`, we don't need the added specificity